### PR TITLE
[SYSTEMML-578] [SYSTEMML-580] Adding ScalaTest for MLPipeline wrappers

### PR DESCRIPTION
**Pull request by niketanpansare**
1. Added ScalaAutomatedTestBase to set appropriate global variables.
2. Added CrossValidator test in LogisticRegressionSuite

### CODE REVIEW

**Comment by niketanpansare** [comment]
@Wenpei Please look into this PR.

**Comment by niketanpansare** [comment]
PLEASE DONOT MERGE UNTIL @Wenpei is done adding fix for MLPipeline CrossValidator example in https://github.com/apache/incubator-systemml/pull/116/

**Comment by dusenberrymw** [comment]
This can be folded into [SYSTEMML-581](https://issues.apache.org/jira/browse/SYSTEMML-581) as well.

File: src/test/scala/org/apache/sysml/api/ml/scala/ScalaAutomatedTestBase.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.apache.sysml.api.ml.scala
+
+import java.io.File
+
+object ScalaAutomatedTestBase {
+  // *** HACK ALERT *** HACK ALERT *** HACK ALERT ***
```
Comment (Wenpei) [inline src/test/scala/org/apache/sysml/api/ml/scala/ScalaAutomatedTestBase.scala]: Some code format issues (table or space ) here.

**Comment by niketanpansare** [comment]
Let's deal with the code formatting later. This is ready to merge after https://github.com/apache/incubator-systemml/pull/116 is merged.

**Comment by niketanpansare** [comment]
Closing this PR.

**Comment by dusenberrymw** [comment]
Just FYI, this can be automatically closed by appending `Closses #117.` to the end of a commit message.  This nice side-effect is that it also links (clickable link) the commit to this PR and the discussion on it.

**Pull request closed without merging**